README.md

```diff
@@ -17,40 +18,97 @@ In contrast, our philosophy is rooted in simplicity:
 
 We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s.
 
-## Usage
+## Quick Start (localnet)
+
+![Demo](https://raw.githubusercontent.com/lambdaclass/ethereum_rust/update-readme/Demo.png)
+
+You'll need docker, docker compose, clang and rust 1.80
```

Comment:
you sure we need all these? I wouldn't reference a particular rust version because otherwise we have to update it every time.

Reply:
You're right about not specifying a rust version. I'm sure we need Docker, not so sure about docker-compose, I think some dependency fails without clang, but I'll double check it and let you know.